<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Termometro</class>
 <widget class="QMainWindow" name="Termometro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Termometro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/Icono</normaloff>:/Icono</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="grpCent">
      <property name="title">
       <string>Centigrados</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdCent">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialCent">
         <property name="minimum">
          <number>-18</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpFahr">
      <property name="title">
       <string>Fahrenheit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="lcdFahr">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 170, 0);
background-color: rgb(85, 255, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialFahr">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>210</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpKelv">
      <property name="title">
       <string>Kelvin</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLCDNumber" name="lcdKelv">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialKelv">
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>372</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialKelv</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdKelv</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
